docs/plus/rbac.md

```diff
@@ -0,0 +1,121 @@
+# Role Based Access Control
+
+Role Based Access Control(RBAC) helps you restrict actions that can be performed on specific Kubernetes resources. To make this possible, RBAC lets you create roles with rules which define access permissions for your specified resource.
+
+These roles can then be binded to Kubernetes subjects, which could be User, Group or ServiceAccount. 
+
+**NOTE:** Rules or permissions are purely additive and there are no deny rules.
```

Comment:
⛏️ This would be a good place to use a  `!!! note` Admonitions

Reply:
Ohh, I did not know that. Nice, will make the update.